e2e/test/scenarios/actions/actions-on-dashboards.cy.spec.js

```diff
@@ -987,6 +992,55 @@ describe(
         });
       });
 
+      it("refetches form values when id changes (metabase#33084)", () => {
+        const actionName = "Update";
+
+        cy.get("@modelId").then(id => {
+          createImplicitAction({
+            kind: "update",
+            model_id: id,
+          });
+        });
+
+        createDashboardWithActionButton({
+          actionName,
+          idFilter: true,
+        });
+
+        filterWidget().click();
+        addWidgetStringFilter("5");
+
+        cy.findByRole("button", { name: actionName }).click();
```

Comment:
This is the same as
```suggestion
        cy.button(actionName).click();
```

while `cy.button` is far more established. 500+ results vs 66 results of `findByRole("button"...` inside `e2e` folder.

Reply:
35990d6777a6694eedb473abc58847de3249d2e7